tests/e2e/paths/search/granules/collection_details/collection_details.spec.js

```diff
@@ -0,0 +1,467 @@
+  expect(gibsProjections).toHaveText(projections)
+}
+
+test.describe('Path /search/granules/collection-details', () => {
+  test.describe('When collection has associated DOIs', () => {
+    test('loads correctly', async ({ page, context }) => {
+      await page.route('**/*.{png,jpg,jpeg}', (route) => route.abort())
+
+      const conceptId = 'C1240222820-ECHO_REST'
+      const cmrHits = 12345
+      const granuleHits = 0
+
+      await page.route(/collections.json/, async (route) => {
+        const query = route.request().postData()
+        expect(query).toEqual('include_facets=v2&include_granule_counts=true&include_has_granules=true&include_tags=edsc.*,opensearch.granule.osdd&page_num=1&page_size=20&sort_key[]=-score')
+
+        await route.fulfill({
+          json: collectionsBody.body,
+          headers: {
+            ...commonHeaders,
+            'cmr-hits': cmrHits.toString()
+          }
+        })
+      })
+
+      await page.route(/granules.json/, async (route) => {
+        const query = route.request().postData()
+        expect(query).toBe('echo_collection_id=C1240222820-ECHO_REST&page_num=1&page_size=20')
+
+        await route.fulfill({
+          json: associatedDoisGranulesBody.body,
+          headers: {
+            ...commonHeaders,
+            'cmr-hits': granuleHits.toString()
+          }
+        })
+      })
+
+      await page.route(/graphql/, async (route) => {
+        const { query } = JSON.parse(route.request().postData())
+
+        if (query === JSON.parse(graphQlGetCollection(conceptId)).query) {
+          await route.fulfill({
+            json: assocatedDoisGraphQlBody,
+            headers: graphQlHeaders
+          })
+        }
+
+        if (query === graphQlGetSubscriptionsQuery) {
+          await route.fulfill({
+            json: getSubscriptionsGraphQlBody,
+            headers: graphQlHeaders
+          })
+        }
+      })
+
+      await page.goto('/search/granules/collection-details?p=C1240222820-ECHO_REST&ee=uat&ac=true')
+
+      // Log-in
+      login(context)
+
+      // Ensure title renders on page correctly
+      testCollectionTitle(page, 'Mapping Example for UMM-C 1.16.1')
+
+      // Ensure short-name, version are present
+      const shortName = 'Mapping Short Name 1.16.1'
+      expect(page.getByTestId('collection-details-header__short-name').filter({ hasText: shortName })).toBeVisible()
+      const version = 'Version 1.16.1'
+      expect(page.getByTestId('collection-details-header__version-id').filter({ hasText: version })).toBeVisible()
+
+      // Ensure that the collections request ocurred and the component is displaying the correct results
+      testCollectionResults(page, cmrHits)
+
+      // Test temporal range
+      testCollectionTemporal(page, '2001-01-01 to 2001-06-01')
+
+      // TODO this is 8 because things are being counted multiple times over
+      testCollectionScienceKeywords(page, 8, [
```

Comment:
Trying to figure this out its just counting these x4 unlike in the cypress one. For some reason playwright is finding more `<li` tags